app/src/main/java/com/infomaniak/mail/ui/main/newMessage/NewMessageViewModel.kt

```diff
@@ -91,12 +91,12 @@ class NewMessageViewModel(application: Application) : AndroidViewModel(applicati
                 true
             }
 
-            if (isSuccess) initUiData()
+            if (isSuccess) populateViewModelWithDraftData()
             emit(isSuccess)
         }
     }
 
-    private fun initUiData() {
+    private fun populateViewModelWithDraftData() {
```

Comment:
We are already in the viewmodel so I find it a bit weird, just replace viewModel with data I find it sufficient, and why `populate` instead of `init`